lib/ansible/plugins/lookup/onepassword_doc.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# (c) 2019, David Taylor <djtaylor13@gmail.com> (onepassword.py used as a starting point)
+# (c) 2018, Scott Buchanan <sbuchanan@ri.pn>
+# (c) 2016, Andrew Zenk <azenk@umn.edu> (lastpass.py used as starting point)
+# (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+    lookup: onepassword_doc
+    author:
+      - David Taylor <djtaylor13@gmail.com>
+      - Scott Buchanan <sbuchanan@ri.pn>
+      - Andrew Zenk <azenk@umn.edu>
+      - Sam Doran<sdoran@redhat.com>
+    version_added: "2.7"
+    requirements:
+      - C(op) 1Password command line utility. See U(https://support.1password.com/command-line/)
+    short_description: fetch documents from 1Password
+    description:
+      - C(onepassword_doc) wraps the C(op) command line utility to fetch documents from 1Password.
+    options:
+      _terms:
+        description: identifier(s) (UUID, name, or subdomain; case-insensitive) of item(s) to retrieve
+        required: True
+      master_password:
+        description: The password used to unlock the specified vault.
+        default: None
+        version_added: '2.7'
```

Comment:
This is new lookup so no need to add `version_added`

Reply:
resolved with e43ffc15aed3df98b611c57a80a606fd287bc20a